test/regress_http.c

```diff
@@ -71,13 +71,37 @@ static struct event_base *exit_base;
 
 static char const BASIC_REQUEST_BODY[] = "This is funny";
 
+/* defines an extended HTTP method "CUSTOM"
+ * without body */
+#define EVHTTP_REQ_CUSTOM	((EVHTTP_REQ_MAX) << 1)
+
+static int ext_method_cb(struct evhttp_extended_method *p)
+{
+	if (p == NULL)
+		return -1;
+	if (p->method) {
+		if(strcmp(p->method, "CUSTOM") == 0) {
```

Comment:
style: missing whitespace and please check the whole patchset.

(we have `checkpatch.sh` and `.clang-format` in the git root, but they can be out dated)

Reply:
hum clang-format-diff is not available on my machine :(

Reply:
ping

Reply:
As said above, `checkpatch.sh` doesn't work on my machine, because` clang-format-diff ` is not available.
Maybe you can run it and send me the results by email.

Reply:
It can use `uncrustify` too.

I cannot do this for you right now, maybe later, but I would suggest you to check this patches manually (or writing your own script) if you cannot use `checkpatch.sh` (actually it requires some modifications since `clang-format-diff` is now `clang-format-diff.py` and has different options style, and also it reports `ENOENT` error to me).

*P.S. I don't think it is a great idea to ask other people to check coding style issues in your patches*

Reply:
I'm sorry. most of the patch if from @errzey anyway... 

I've run `Uncrustify-0.68_f` on `http.c`, it makes me a lot of differences. Some of them are a bit strange.
```
@@ -4923,7 +4942,8 @@
                case PART_FRAGMENT:
                        cp += strlen(cp);
                        break;
-               };
+               }
+               ;
                return cp;
        }
```
I'll check all changes that are related to this PR and correct.

Reply:
```
$ git show 29ce63b19d0cf1e0ca07ed3ca3fed1fc9e89039a | ./checkpatch.sh -U -p
Not implemented
$ ./checkpatch.sh -U -r 29ce63b19d0cf1e0ca07ed3ca3fed1fc9e89039a
$ ./checkpatch.sh -U -d http.c
[... patch ...]
```

Reply:
>I've run Uncrustify-0.68_f on http.c, it makes me a lot of differences

It is a bad idea to run uncrustify on the whole file, since the coding style in it vary

>Some of them are a bit strange.

Indeed, so basically just run it only on you patch

>$ git show 29ce63b19d0cf1e0ca07ed3ca3fed1fc9e89039a | ./checkpatch.sh -U -p
>Not implemented

But I guess it is not possible to run it only on the patch right now, egh.

So I would suggest you to look through the patch manually, this should take too much time, since the patch is pretty small

Reply:
I have a problem. you said : 

> style:  `const char *foo()` over `const char * foo()` (i.e. for functions the same as for variable definitions, e.t.c.)

but uncrustify outputs 
```
-static const char *evhttp_method_(struct evhttp_connection *evcon,
-       enum evhttp_cmd_type type, ev_uint16_t *flags);
+static const char * evhttp_method_(struct evhttp_connection *evcon,
+                                  enum evhttp_cmd_type type, ev_uint16_t *flags);
```
I think it is because of this : 
https://github.com/libevent/libevent/blob/master/.uncrustify#L15
```
sp_after_ptr_star                        = remove
sp_after_ptr_star_func                   = add
```

should I fix `.uncustify` ?

Reply:
The basic rule is: use coding style like in the code around

>should I fix .uncustify ?

Maybe it is a good idea, but I need to check, since AFAIR I created this config to get minimal changes to the current sources